src/components/backend-ai-resource-monitor.ts

```diff
@@ -507,8 +506,18 @@ export default class BackendAiResourceMonitor extends BackendAIPage {
     }
   }
 
-  async _refreshResourcePolicy() {
-    return this.resourceBroker._refreshResourcePolicy().then(() => {
+  /**
+   *  If bot refreshOnly and active are true, refresh resource monitor indicator
+   * 
+   * @param {boolean} refreshOnly 
+   * 
+   */
+  _refreshResourcePolicy(refreshOnly = false) {
```

Comment:
If you remove async here, `await` will not work on other references. See below.

Reply:
Oops, My mistakes. I'll fix it right away.